src/backend/assets/js/main.js

```diff
@@ -82,38 +82,80 @@ const loadVideoAndSlides = function(div, videoURL, slideURL) {
     const faviconDiv = $('#v' + div);
 
     if (videoURL.indexOf('v=') !== -1) {
-      video = videoURL.split('v=')[1].replace(/[&#].*/, '');
+        video = videoURL.split('v=')[1].replace(/[&#].*/, '');
```

Comment:
The indentation should remain constant in every file of the project. Right now it is `2 spaces`.

Reply:
I think we can let a linter handle this one